README.md

```diff
@@ -23,7 +23,7 @@ We're using the following stack:
 
 - [Jest](https://jestjs.io/)
 - [Prettier](https://prettier.io/)
-- [Lerna](https://lerna.js.org/)
+- [Lerna](https://github.com/lerna/lerna/tree/main/commands/add#readme)
```

Comment:
I think the original link is more appropriate here

Reply:
I noticed the website itself doesn't contain much info for commands as `lerna add ...`
But yeah if I put the link at the other spot in the documentation sounds good. 
Thanks!